core/src/main/java/io/opentelemetry/android/features/diskbuffering/scheduler/DefaultExportScheduler.kt

```diff
@@ -15,14 +15,11 @@ import java.util.concurrent.TimeUnit
 
 class DefaultExportScheduler(
     periodicWorkProvider: () -> PeriodicWork,
+    private val exportScheduleDelayMillis: Long = TimeUnit.SECONDS.toMillis(10),
```

Comment:
@breedx-splk do we use `@JVMOverloads constructor` to have nicer api and good backward compatibility support in case of default param? If yes we should add the same here